src/Umbraco.Forms.Integrations.Crm.Hubspot/Services/HubspotContactService.cs

```diff
@@ -194,7 +199,12 @@ public async Task<CommandResult> PostContactAsync(Record record, List<MappedProp
                 // A 409 - Conflict response indicates that the contact (by email address) already exists.
                 if (response.StatusCode == HttpStatusCode.Conflict)
                 {
-                    return await UpdateContactAsync(record, authenticationDetails, propertiesRequestV1, emailValue);
+                    if (!_settings.AllowContactUpdate)
+                        _logger.Error<HubspotContactService>("Hubspot contact update is not allowed.");
+
+                    return _settings.AllowContactUpdate 
+                        ? await UpdateContactAsync(record, authenticationDetails, propertiesRequestV1, emailValue)
+                        : CommandResult.Failed;
```

Comment:
```suggestion
                        : CommandResult.Success;
```

Workflow has still succeeded if we've skipped updates due to configuration.